app/models/document.rb

```diff
@@ -239,7 +239,7 @@ def publish!
         email_alert_api.send_alert(EmailAlertPresenter.new(self).to_json)
       end
 
-      publish_request.code == 200 && rummager_request.code == 200
+      publish_request.code == 200 && rummager_request.code == 202
```

Comment:
Yes, feels very brittle. It would be better to rely on the exceptions here (assuming they're raised).